Pedro/tinkering (#491)
AequilibraE/aequilibrae: README.md

NumPY arrays, making it the perfect candidate for use-cases where transport is 
one component of a bigger and more general planning or otherwise analytical 
modeling pipeline.

Different than in traditional packages, AequilibraE's network is stored in 
SQLite/Spatialite, a widely supported open format, and its editing capabilities
are built into its data layer through a series of spatial database triggers, 
which allows network editing to be done on Any GIS package supporting SpatiaLite, 
through a dedicated Python API or directly from an SQL console while maintaining
full geographical consistency between links and nodes, as well as data integrity
and consistency with other model tables.

AequilibraE provides full support for OMX matrices, which can be used as input
for any AequilibraE procedure, and makes its outputs, particularly skim matrices 
readily available to other modeling activities.

AequilibraE includes multi-class user-equilibrium assignment with full support
for class-specific networks, value-of-time and generalized cost functions, and 
includes a range of equilibration algorithms, including MSA, the traditional 

> Frank-Wolfe the state-of-the-art Bi-conjugate Frank-Wolfe.

Reviewer: ```suggestion
Frank-Wolfe as well as the state-of-the-art Bi-conjugate Frank-Wolfe.
```

AequilibraE's support for public transport includes a GTFS importer that can 
map-match routes into the model network and an optimized version of the

> traditional Optimal-Strategies transit assignment, although AequilibraE's data 
> model in disk provides full support for schedule-based assignments in the future.

Reviewer: ```suggestion
traditional "Optimal-Strategies" transit assignment, and full support in the data 
model for other schedule-based assignments to be implemented in the future.
```
